docs/getting-started/deploy-incrementor-to-testnet.mdx

```diff
@@ -0,0 +1,87 @@
+---
+sidebar_position: 5
+title: 4. Deploy Incrementor to Testnet
+description: Deploy the Incrementor contract to Testnet.
+---
+
+<head>
+  <title>Write a smart contract that stores and retrieves data.</title>
```

Comment:
```suggestion
  <title>4. Deploy Incrementor to Testnet.</title>
```

Do we need this at all, though? I haven't seen the `<head>` tag on other pages. Nice feature, when it's needed, but it seems like maybe we don't need to customize the defaults on this page.

Reply:
Oh, it looks like it is on the other pages now. Huh.

Reply:
I agree that maybe it's not 100% necessary, but just added it to be consistent with the other pages. I'm happy to rethink/remove this as we continue to refine these pages.